Tests/SwiftFormatPrettyPrintTests/TernaryExprTests.swift

```diff
@@ -43,10 +43,10 @@ public class TernaryExprTests: PrettyPrintTestCase {
       """
       let someLocalizedText =
         shouldUseTheFirstOption
-          ? stringFunc(name: "Name1", comment: "short comment")
-          : stringFunc(
-            name: "Name2", comment: "Some very, extremely long comment",
-            details: "Another comment")
+        ? stringFunc(name: "Name1", comment: "short comment")
+        : stringFunc(
```

Comment:
Should the lines starting with "?" and ":" (the choices of the ternary) be indented +1 continuation from the condition here? Seems like that might be nice to show that it's a continuation of ternary expr.

If that's a change that we want, I think it's fine to fix in a later PR.

Reply:
I thought about that for a bit too when I saw this change and I wasn't sure which way I wanted to go with it. I think I'll need to see what happens when I start fixing nested ternaries to see where things land. On the one hand, the continuation lines start with operators so it's clear they're continuations, but on the other hand a ternary has more "structure" than regular operators and showing that structure through indentation might also be good...